src/Microsoft.Diagnostics.Monitoring.WebApi/Metrics/JsonCounterLogger.cs

```diff
@@ -48,16 +59,11 @@ protected override async Task SerializeAsync(ICounterPayload counter)
                 writer.WriteString("unit", counter.Unit);
                 writer.WriteString("counterType", counter.CounterType.ToString());
 
+                writer.WriteString("tags", counter.Metadata);
+
                 //Some versions of .Net return invalid metric numbers. See https://github.com/dotnet/runtime/pull/46938
                 writer.WriteNumber("value", double.IsNaN(counter.Value) ? 0.0 : counter.Value);
 
-                writer.WriteStartObject("metadata");
```

Comment:
This is likely a breaking change; need to convert to dictionary using new api from Monitoring.EventPipe

Reply:
From our PR review meeting, it sounds like this won't be a breaking change since this functionality didn't ship for 7.0 GA.